chromium_src/components/translate/core/browser/translate_language_list.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_TRANSLATE_CORE_BROWSER_TRANSLATE_LANGUAGE_LIST_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_TRANSLATE_CORE_BROWSER_TRANSLATE_LANGUAGE_LIST_H_
+
+namespace translate {
+class TranslateLanguageList;
+using BraveTranslateLanguageList = TranslateLanguageList;
```

Comment:
pls keep class name prefix the same as in chromium. only append to the class name, don't prepend.
`TranslateLanguageList_BraveImpl`

Reply:
Done